### PR TITLE
Examples to spawn sdf files

### PR DESCRIPTION
**Pull request by fibonatic**
This adds files which spawns sdf files from a list. The list is specified in the yaml file and is spawned by the python file in the scripts folder. This script uses src/ed_object_models/spawn_sdf_in_gazebo.py from the ed_object_models repository, which currently only exists on the spawn_sdf branch.

### CODE REVIEW

File: scripts/spawn_sdf_test.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/python
+from ed_object_models import spawn_sdf_in_gazebo
```
Comment (MatthijsBurgh) [inline scripts/spawn_sdf_test.py:2]: If it is a test put it in the test folder.

File: scripts/spawn_sdf_test.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+if __name__ == "__main__":
+    # This spawns 7 coke cans, similar to fast_simulator_data/worlds/robotics_testlabs/robotics_testlabs.py
+    spawn_sdf_in_gazebo.from_yaml('/sdf_test/objects_list_test.yaml')
```
Comment (MatthijsBurgh) [inline scripts/spawn_sdf_test.py]: This looks like an absolute path to me. Which will not work. Best solution is to use rospack to find the package. Or have the path as an argument.

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]

File: scripts/spawn_sdf_test.py (hunk 1 above)
Comment (fibonatic) [reply scripts/spawn_sdf_test.py:2]: It is more of a simple example of how to use it. Maybe put it into (currently not existing) readme file for this package? Or would a test folder still be the better option?

File: scripts/spawn_sdf_test.py (hunk 2 above)
Comment (fibonatic) [reply scripts/spawn_sdf_test.py]: I changed the functionality of the function, which now requires the full yaml file path and updated this code accordingly.

File: scripts/spawn_sdf_test.py [hunk 3]
```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/python
 from ed_object_models import spawn_sdf_in_gazebo
+from rospkg import RosPack
 
 if __name__ == "__main__":
     # This spawns 7 coke cans, similar to fast_simulator_data/worlds/robotics_testlabs/robotics_testlabs.py
-    spawn_sdf_in_gazebo.from_yaml('/sdf_test/objects_list_test.yaml')
+    package_path = RosPack().get_path('fast_simulator_data')
```
Comment (MatthijsBurgh) [inline scripts/spawn_sdf_test.py:8]: This requires roslib as a run dependency

File: scripts/spawn_sdf_test.py (hunk 1 above)
Comment (MatthijsBurgh) [reply scripts/spawn_sdf_test.py:2]: If this keeps a test, put it in the test folder. Also would be nice to have this feature fully implemented for robotics_testlabs incl. all environment variables, launch files etc.

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]

File: sdf_test/objects_list_test.yaml [hunk 4]
```diff
@@ -1,7 +1,7 @@
-- {id: "coke-1", type: "coke_can", x: 3.196, y: 4.652, z: 0.87, roll: 0, pitch: 1.57, yaw: 0}
-- {id: "coke-2", type: "coke_can", x: 3.214, y: 4.050, z: 0.87, roll: 0, pitch: 0, yaw: 0.2}
-- {id: "coke-3", type: "coke_can", x: 3.463, y: 4.353, z: 0.87, roll: 0, pitch: 0, yaw: 0.4}
-- {id: "coke-4", type: "coke_can", x: 1.056, y: 3.471, z: 0.8 , roll: 0, pitch: 0, yaw: 0.6}
-- {id: "coke-5", type: "coke_can", x: 0.676, y: 3.559, z: 0.8 , roll: 0, pitch: 0, yaw: 0.8}
```
Comment (MatthijsBurgh) [inline sdf_test/objects_list_test.yaml]: These are round, so we will not see that. But good to test it. Use other objects if possible.

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**